packages/agoric-cli/package.json

```diff
@@ -34,6 +34,7 @@
     "@agoric/casting": "^0.3.2",
     "@agoric/cosmic-proto": "^0.2.1",
     "@agoric/ertp": "^0.15.3",
+    "@agoric/inter-protocol": "^0.13.1",
```

Comment:
i was reluctant to add this dependency, but the agops CLI commands do depend on Inter Protocol, whether they were importing it or not